progenitor-client/src/progenitor_client.rs

```diff
@@ -386,7 +386,7 @@ const PATH_SET: &percent_encoding::AsciiSet = &percent_encoding::CONTROLS
     .add(b'/')
     .add(b'%');
 
-#[doc(hidden)]
+/// Percent encode input string.
```

Comment:
why doc this? this is really only for internal use.

Reply:
It is part of the public API.

Reply:
Ok: I don't want to document this. I don't want people using it explicitly.

Reply:
ok, ill revert this bit